packages/cli/src/commands/exchange/stable.ts

```diff
@@ -0,0 +1,29 @@
+import { StableToken } from '@celo/contractkit'
+import { flags } from '@oclif/command'
+import ExchangeStableBase from '../../exchange-stable-base'
+import { enumEntriesDupWithLowercase } from '../../utils/helpers'
+
+const stableTokenOptions = enumEntriesDupWithLowercase(Object.entries(StableToken))
+export default class ExchangeStable extends ExchangeStableBase {
+  static description = 'Exchange Stable Token for CELO via the stability mechanism'
+
+  static flags = {
+    ...ExchangeStableBase.flags,
+    stableToken: flags.enum({
+      options: Object.keys(stableTokenOptions),
+      description: 'Name of the stable token to be transfered',
+    }),
+  }
+
+  static examples = [
+    'stable --value 10000000000000 --from 0xc1912fEE45d61C87Cc5EA59DaE31190FFFFf232d --stableToken cUSD',
```

Comment:
Since this is an example for any stable token, what do you think about `stableTokenSymbol` or `cStableTokenSymbol` instead of `cUSD`? 

current:
` 'stable --value 10000000000000 --from 0xc1912fEE45d61C87Cc5EA59DaE31190FFFFf232d --stableToken cUSD'`

suggested:
` 'stable --value 10000000000000 --from 0xc1912fEE45d61C87Cc5EA59DaE31190FFFFf232d --stableToken cStableTokenSymbol'`

Reply:
I added that as an example with real data, but yes, I can change it